synchronization/src/androidTest/java/de/cyface/synchronization/SetAccountFlagTest.java

```diff
@@ -134,8 +135,11 @@ public void tearDown() {
      * This bug was only reproducible in integration environment (device and emulator) but not as robolectric test.
      * <p>
      * This test may be flaky on a <b>real</b> device when the network changes during the test.
+     * <p>
+     * This test is flaky on the Github CI emulator, but work on local emulators [STAD-425].
      */
     @Test
+    @FlakyTest
```

Comment:
My main goal in this PR is to get as many of the connected tests stable on the CI again on SDK 6 without having to invest too much time.

Marking this single test as flaky would be a good compromise and a better starting point than the Bitrise CI which was very hacky and unstable for a  longer time.